files/zh-cn/web/api/element/firstelementchild/index.md

```diff
@@ -66,7 +66,5 @@ console.log(foo.firstElementChild.textContent);
 
 ## 参见
 
-## [Ed](/zh-CN/docs/Web/API/Element/lastElementChild$edit#参见)
```

Comment:
My original edit was bad, but I think the link was supposed to be in the section if i look at other languages

Reply:
It's still a good PR, thanks for your help.